src/forking-store.js

```diff
@@ -10,62 +10,15 @@ import {
 
 const BASE_GRAPH_STRING = "http://mu.semte.ch/libraries/rdf-store";
 
-/**
```

Comment:
Moved below because it makes the file easier to read (imo) since the ForkingStore is the primary subject here.